zos_concepts/gdg_datasets/create_copy_submit/README.md

```diff
@@ -0,0 +1,87 @@
+# Create GDG, copy delete and submit of Generation Data Group
+This playbook demonstrates how to create a GDG create copy delete and submit
```

Comment:
This playbook seems to have no Edit'ing portion, my suggestion would have been to use the non-gdg example which did include editing and you only need to add GDG support. 

What we are trying to demonstrate is GDG support, not a new use case really, so copying a previous one is a easy way to do that.

Reply:
There is not edit portion